pkg/bandwidth/bandwidth.go

```diff
@@ -55,6 +49,15 @@ func ProbeBandwidthManager() {
 			kernelGood = true
 		}
 	}
+	option.Config.ResetQueueMapping = kernelGood
```

Comment:
Is this bound by the >5.1 kernel requirement? We can clear the queue mapping on older kernels as long as the BPF programs have access to the `queue_mapping` field, right?

Reply:
Yes, from bpf side it's not writable but only readable on <= 5.1.

(https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=74e31ca850c1cddeca03503171dd145b6ce293b6)

Reply:
For even older kernels, I'd suggest to wait until https://patchwork.kernel.org/project/netdevbpf/patch/ef4cf3168907944502c81d8bf45e24eea1061e47.1641427152.git.daniel@iogearbox.net/ gets in and backported.